src/js/claims-status/containers/AppealInfo.jsx

```diff
@@ -82,17 +103,17 @@ export class AppealInfo extends React.Component {
       // Yes, we have your appeals. No, the one you requested isn't one of them.
       appealContent = <AppealNotFound/>;
     } else if (appealsAvailability === USER_FORBIDDEN_ERROR) {
-      appealContent = <h1>We couldn’t find your records</h1>;
+      appealContent = appealsDownMessage;
     } else if (appealsAvailability === RECORD_NOT_FOUND_ERROR) {
-      appealContent = <h1>We couldn’t find your records</h1>;
+      appealContent = recordsNotFoundMessage;
     } else if (appealsAvailability === VALIDATION_ERROR) {
-      appealContent = systemDownMessage;
+      appealContent = appealsDownMessage;
     } else if (appealsAvailability === BACKEND_SERVICE_ERROR) {
-      appealContent = systemDownMessage;
+      appealContent = appealsDownMessage;
     } else if (appealsAvailability === FETCH_APPEALS_ERROR) {
-      appealContent = systemDownMessage;
+      appealContent = appealsDownMessage;
     } else {
-      appealContent = systemDownMessage;
+      appealContent = appealsDownMessage;
```

Comment:
@gnakm Verifying: We don't actually want the generic system down message to appear at all, is that correct? We want to use this new messaging whenever something goes wrong?

Reply:
Yes!

Reply:
Yass please.